node_test.go

```diff
@@ -529,6 +525,27 @@ func TestCidAndBigInt(t *testing.T) {
 	}
 }
 
+func TestEmptyCid(t *testing.T) {
+	type Foo struct {
+		A cid.Cid
+	}
+	type Bar struct {
+		A cid.Cid `refmt:",omitempty"`
```

Comment:
The semantics without omitempty are surprising. I get that omitempty skips encoding zero values. But without omitempty you can't encode a zero value? What's the intent of semantics like that and what if I really, truly want an empty value?